core/src/test/resources/sql_schema/check_in.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE X
+(
+  ID                 UUID PRIMARY KEY,
+  F_ID               TEXT NOT NULL,
+  P_AT               TIMESTAMP,
+  STATUS             TEXT NOT NULL,
+  CREATED_AT         TIMESTAMP NOT NULL,
+  CREATED_BY         TEXT NOT NULL,
+  LAST_UPDATED_AT    TIMESTAMP,
+  LAST_UPDATED_BY    TEXT,
+  W_ID               TEXT NOT NULL,
+  EXPR_DATE          DATE NOT NULL
+);
+
+ALTER TABLE X ADD CONSTRAINT check_status
+  CHECK (STATUS in ('A', 'B'));
+
+
```

Comment:
could also make sense to have a case in which the `CHECK` is directly in the `CREATE` and not just the `ALTER`. I would assume that the databases would treat them exactly the same, but can never be sure :)

Reply:
Makes sense. I added a new test to confirm that the enum constraint is also extracted when declared within a CREATE TABLE statement.